script.js

```diff
@@ -0,0 +1,22 @@
+const form = document.getElementById('form');
+const password1El = document.getElementById('passwoed1');
+const password2El = document.getElementById('passwoed2');
+const messageContainer = document.querySelector('.message-container');
+const message =document.getElementById('message');
+
+let isValid = false;
+function validateForm () {
+    isValid = form.checkValidity();
+    message.textContent = 'Please fill out all fields.';
+    message.style.color = 'red';
+    messageContainer.style.borderColor = 'red';
+}
+function processFormData (e) {
+    e.preventDefault();
```

Comment:
nice catch here to prevent the browser default behaviour.
just out of  curiosity; why do you think the default behaviour of the browser when submit a form it reload?  can you search and let me know!